sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities_async.py

```diff
@@ -597,6 +598,32 @@ async def test_phi_domain_filter(self, client):
         self.assertEqual(phone.category, "PhoneNumber")
         self.assertEqual(microsoft.category, "Organization")
 
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_categories_filter(self, client):
+        result = await client.recognize_pii_entities(
+            ["My name is Inigo Montoya, my SSN in 243-56-0987 and my phone number is 333-3333."],
+            categories_filter=[PiiEntityCategoryType.US_SOCIAL_SECURITY_NUMBER]
+        )
+
+        self.assertEqual(len(result[0].entities), 1)
+        entity = result[0].entities[0]
+        self.assertEqual(entity.category, PiiEntityCategoryType.US_SOCIAL_SECURITY_NUMBER.value)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_categories_filter_with_domain_filter(self, client):
+        # Currently there seems to be no effective difference with or without the PHI domain filter.
+        result = await client.recognize_pii_entities(
+            ["My name is Inigo Montoya, my SSN in 243-56-0987 and my phone number is 333-3333."],
+            categories_filter=[PiiEntityCategoryType.US_SOCIAL_SECURITY_NUMBER],
+            domain_filter=PiiEntityDomainType.PROTECTED_HEALTH_INFORMATION
+        )
+
+        self.assertEqual(len(result[0].entities), 1)
+        entity = result[0].entities[0]
+        self.assertEqual(entity.category, PiiEntityCategoryType.US_SOCIAL_SECURITY_NUMBER.value)
```

Comment:
test look great, and I think you've already mentioned this offline. But just for posterity, can you also add tests for `pii-categories` to the analyze endpoint? thanks!